### DISCUSSION TITLE
Import records

### DISCUSSION THREAD

**Discussion by viezel** [filamentphp/filament#108] [Ideas]
Most admin panels gives the user the ability to import data. 

1. Upload: user Uploads an excel/csv 
2. Review/validate: User maps imported data to columns. System reports any validation concerns
3. importing: Start import with progress bar
4. Summary: including failed records to import

Would be a nice feature here.

**Comment by robertopc** [comment]
This is a really useful thing.

**Comment by ryangjchandler** [comment]
Given the scope and size of such a feature, I'd say it's best kept as a plugin.
